src/app/containers/FeatureIndex/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```

Comment:
Each pageType ’container’ looks to be exactly the same apart from the component it renders (and the propTypes expected). To reduce repeated logic and tests could this be one component that accepts data and renders the pageType requested?

Reply:
You are right, they are identical apart from the rendered component. It would make sense to have a main container for both.

Reply:
Ah I meant more that the `Article`, `FrontPage`, `RadioPage` etc... 'Containers' are exactly the same component, that import the 5 HOCs and export a 'Main' component. This could be one HOC that wraps each 'Main' component?